### DISCUSSION TITLE
Flags del Z80

### DISCUSSION THREAD

**Discussion by SirRickster** [gusmanb/ZXBasicStudio#29] [Ideas]
Que te parece este añadido a los registros? Ya casi lo tengo listo, aunque se me está resistiendo el Binding del componente, por mucho que hago un update() no me actualiza el contenido. A ver si doy con el motivo.

![imagen](https://github.com/gusmanb/ZXBasicStudio/assets/148114324/6bff343b-d4da-4c22-bbb7-2d03bed1d2d6)

El resultado tendría que ser algo así una vez actualizado

![imagen](https://github.com/gusmanb/ZXBasicStudio/assets/148114324/b5d15073-195d-4935-b1a6-f5a86a31622f)

Y con su tooltip de ayuda

![imagen](https://github.com/gusmanb/ZXBasicStudio/assets/148114324/a3e1d2c4-f9ff-4def-afee-9d2cb515bad0)

**Comment by gusmanb** [comment]
Ole!! Eso está de narices!
